Nicoconte/Coworking-MusicHub-Server
# Dev

User ft such as:
 - User registration
 - User Login
 - Validation by Token
 
Room ft. such as :
 - CRUD Room 
 - Add Participant
 - Check validate private room by password
 - Create invite code
 - Get room by invite code
 - List public and user room
